storehaus-postgresql/src/main/scala/com/twitter/storehaus/postgres/Read.scala

```diff
@@ -0,0 +1,15 @@
+package com.twitter.storehaus.postgres
+
+
+/**
+  * Type class for deserializing PostgreSQL native data to scala types
+  *
+  * Created by ponkin on 10/20/16.
+  */
+trait Read[A] {
```

Comment:
you could do in your package:
```scala
type PostgresValueConverter[A] = Injection[A, PostgresValue]
```
then in the `object PostgresValue { ...` define the implicits and it should be pretty clean and pick them up automatically.